blackbox/docs/best_practice/systables.txt

```diff
@@ -76,9 +76,9 @@ and **sys.operations_log**, respectively.
 Step 3: Analyzing Cluster Resources
 ===================================
 
-Sometimes it’s not a single query that causes problems, but a component of your
+Sometimes it's not a single query that causes problems, but a component of your
 distributed cluster. To find out more about it, check the table
-**sys.cluster**, which holds a single row containing the name and id of the
+**sys.cluster**, which holds a single row containing the name and ID of the
```

Comment:
because "ID" is an abbreviation and should always be capitalised in prose (it's okay to be lowercase as a literal, column name, variables, etc)

I will add this to our RST standards doc

https://github.com/crate/team/blob/master/technical_writing/rst_standards.md